### PR TITLE
Add support for jalali months in persian alphabets in strftime

### PR DESCRIPTION
**Pull request by MojixCoder**
Now we have `%B` which gives us `Azar` for example.  

I've added `%E` which gives us `آذر‍` in this case.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## [3.7.1] - 2021-12-21
```
Comment (hramezani) [inline CHANGELOG.md:3]: we don't need to define the version and date here we can determine them in release time.

```suggestion
## Unreleased
```

File: setup.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 setup(
     name='jdatetime',
-    version='3.7.0',
+    version='3.7.1',
```
Comment (hramezani) [inline setup.py:7]: we don't need this change in this patch

Comment (MojixCoder) [reply setup.py]: Done.

File: CHANGELOG.md (hunk 1 above)
Comment (MojixCoder) [reply CHANGELOG.md]: Commited.

File: t/test.py [hunk 3]
```diff
@@ -89,6 +89,14 @@ def test_init_accepts_instance_locale(self):
         date = jdatetime.date(1397, 4, 23, locale=jdatetime.FA_LOCALE)
         self.assertEqual(date.strftime('%A'), u'شنبه')
 
+    def test_strftime_fa_month(self):
+        j_months_fa = jdatetime.date.j_months_fa
+        month = 0
+        for date in range(12):
+            month += 1
+            j_month_fa = jdatetime.date(year=1400, month=month, day=10).strftime("%E")
+            self.assertEqual(j_month_fa, j_months_fa[month - 1])
```
Comment (hramezani) [inline t/test.py:98]: I would make it simpler:
```suggestion
        for month in range(1, 13):
            self.assertEqual(
                jdatetime.date(year=1400, month=month, day=10).strftime("%E"),
                jdatetime.date.j_months_fa[month - 1],
            )
```

Comment (MojixCoder) [reply t/test.py]: Thanks for the review, changes have been applied.

**Review by hramezani — APPROVED** [review]
Thanks @MojixCoder for the patch :+1:

**Comment by MojixCoder** [comment]
I have worked on the `change str.replace` TODO.  
But yet I don't wanna push because it will be mixed with this pull request if I push my changes.  
Is this PR gonna be merged soon? or is it possible to open two pull requests at the same time?

**Comment by hramezani** [comment]
> I have worked on the change str.replace TODO (in strftime method).

Good to hear!

>But I don't wanna push yet because it will be mixed with this pull request if I push my changes.
Is this PR gonna be merged soon? or is it possible to open two pull requests at the same time?


You can create a new PR for that change. Don't add that change to this patch.

**Comment by slashmili** [comment]
Thanks for your contribution.

I couldn't find `%E` directive in [datetime](https://docs.python.org/3/library/datetime.html) docs. is it described in other docs `%E`? We avoid adding additional directive to be compatible with the official module.

**Comment by MojixCoder** [comment]
I think no we don't have since python doesn't have jalali dates.

**Comment by hramezani** [comment]
@MojixCoder it seems that we can get the month in Persian by setting `locale` in object creation.
Thanks @slashmili for clarification.

**Comment by MojixCoder** [comment]
Yes we can do it, But for example in `django-jalali` when we are using jalali template tags. It's not that easy to set `locale` in templates. and it's also very common.

**Comment by slashmili** [comment]
just for sake of prosperity we discussed it in #98 for Django case, you need to set it in as `LANGUAGE_CODE` setting.

**Pull request closed without merging**